src/screens/Upload/CameraScreen.tsx

```diff
@@ -3,27 +3,42 @@ import * as ImagePicker from 'expo-image-picker';
 import firebase from 'firebase/app';
 import React, { useEffect, useState } from 'react';
 import {
-  Button,
   StatusBar,
   LogBox,
```

Comment:
i allowed access to all photos + camera but when trying to takes photos received error: missing camera or camera roll permissions
![image](https://user-images.githubusercontent.com/57604304/156951060-12f7d9d3-8afc-4f0a-b72c-ab48f6e211d4.png)
possible fix: https://github.com/expo/expo/issues/4826 using expo-camera package?